firebase-firestore/src/androidTest/java/com/google/firebase/firestore/testutil/IntegrationTestUtil.java

```diff
@@ -125,6 +128,10 @@ public enum TargetBackend {
 
   private static boolean backendPrimed = false;
 
+  private static final Random RANDOM = new Random();
+
+  private static final int MAX_BYTES_PER_FIELD_VALUE = 1048487;
```

Comment:
I'm also on the fence about moving this into `getLargestDocContent()`. On the one hand, literal constants are usually kept outside the method scope. On the other hand, there are no other usages of it. Leaning a bit towards keeping it here.